src/frontend/displaywindow/clexiconreadwindow.h

```diff
@@ -26,6 +26,8 @@ class CSwordModuleInfo;
 /** \brief The class used to display lexicons. */
 class CLexiconReadWindow: public CDisplayWindow {
 
+    Q_OBJECT
+
```

Comment:
This was already added by commit c394c367cce878cd5a1caa478e3008d266437c51 (less elegantly, I know).